packages/dd-trace/src/profiling/loggers/composite.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+/* eslint-disable no-console */
+
+class CompositeLogger {
```

Comment:
There appears to be only one non-composite logger type. Why have CompositeLogger at all? ConsoleLogger doesn't take in a Console instance, so there can only ever be one logger, right?

Reply:
The configuration option accepts an array of loggers, which must then be mixed somehow, so I added this class. Right now it's not exposed since most of the public API was removed for the merge.